doc/dev_guide/internals/box_protocol.rst

```diff
@@ -1406,6 +1413,35 @@ IPROTO_FLAG_COMMIT (0x01) will be set if this is the last message for a transact
 IPROTO_FLAG_WAIT_SYNC (0x02) will be set if this is the last message for a transaction which cannot be completed immediately,
 IPROTO_FLAG_WAIT_ACK (0x04) will be set if this is the last message for a synchronous transaction.
 
+..  _box_protocol-raft:
+
+IPROTO_RAFT = 0x1e
+~~~~~~~~~~~~~~~~~~
+
+The IPROTO_RAFT request is broadcasted to all the replicas connected to the node.
```

Comment:
I would add that it's broadcasted not randomly, but once this node's raft state changes. For example, it starts new elections, or bumps the term, or maybe votes for someone. Or even becomes the leader.

Reply:
Corrected